app/controllers/home_controller.rb

```diff
@@ -8,30 +8,33 @@ def index
   end
 
   def check_card
-    @card = Card.find(check_params[:card_id])
-    result = @card.check_translation(check_params[:user_translation],
-                                     check_params[:response_time])
-    case result[:assession_mark]
-    when 5 then
-      flash[:success] = t('card.check_success')
-    when 4, 3 then
-      flash[:warning] = t('card.check_delay')
-    when 2 then
-      flash[:warning] = t('card.typos',
-                          correct_answer: @card.original_text,
-                          typed_word: check_params[:user_translation])
-    else
-      flash[:warning] = t('card.check_error',
-                          right_card: @card.original_text)
+    card = Card.find(check_params[:card_id])
+    result = card.check_translation(check_params[:user_translation],
+                                    check_params[:response_time].to_i)
+    @response_alert = {}
+    respond_to do |format|
+      case result[:assession_mark]
+      when 5 then
+        @response_alert[:success] = t('card.check_success')
+      when 4, 3 then
+        @response_alert[:info] = t('card.check_delay')
+      when 2 then
+        @response_alert[:warning] = t('card.typos',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.